dragonchain/webserver/lib/verifications.py

```diff
@@ -50,6 +52,42 @@ def query_verifications_v1(block_id: str, params: Optional[Dict[str, Any]] = Non
     return _get_verification_records(block_id, level)
 
 
+def query_subsequent_interchain_broadcasts_v1(block_id: str) -> List[Any]:
+    """Return the subsequent broadcasts to other L5 networks"""
+    _log.info(f"Getting subsequent L5 verifications for {block_id}")
+    results = []
+    l5_verifications = _get_verification_records(block_id, 5)
+    if len(l5_verifications) > 0:
+        timestamp = cast(List[Any], l5_verifications)[0]["header"]["timestamp"]
+        l5_nodes = redisearch._get_redisearch_index_client(redisearch.Indexes.verification.value).redis.smembers(redisearch.L5_NODES)
```

Comment:
why are you getting a set of L5 nodes and doing N queries?
Isn't the whole point of the verification index that they're _only_ L5 blocks anyways? It seems like you would only need one query if that were the case.